src/TransactionFormatter.java

```diff
@@ -0,0 +1,38 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class TransactionFormatter {
+
+    Currency currency = new Currency();
+
+    public String[] format(Transaction[] transactions) {
```

Comment:
I don't think this method is tested nor necessary. We can do the loop outside of this class and only have one `public` method.